docs/docsite/rst/contributor_guide.rst

```diff
@@ -0,0 +1,222 @@
+.. _ansible_collections.community.hashi_vault.docsite.contributor_guide:
+
+*****************
+Contributor guide
+*****************
+
+This guide aims to help PR authors contribute to the ``community.hashi_vault`` collection.
+
+.. note::
+
+	This guide is a work-in-progress and should not be considered complete. Check back often as we fill out more details based on experience and feedback, and please let us know how this guide can be improved by opening a `GitHub issue in the repository <https://github.com/ansible-collections/community.hashi_vault/issues>`_.
+
+
+.. contents::
+  :local:
+  :depth: 3
+
+
+Quick start
+===========
+
+#. Log into your GitHub account.
+#. Fork the `ansible-collections/community.hashi_vault repository <https://github.com/ansible-collections/community.hashi_vault>`_ by clicking the **Fork** button in the upper right corner. This will create a fork in your own account.
+#. Clone the repository locally, following :ref:`the example instructions here <hacking_collections>` (but replace ``general`` with ``hashi_vault``). **Pay special attention to the local path structure** of the cloned repository as described in those instructions (for example ``ansible_collections/community/hashi_vault``).
+#. As mentioned on that page, commit your changes to a branch, push them to your fork, and create a pull request (GitHub will automatically prompt you to do so when you look at your repository).
+#. :ref:`See the guidance on Changelogs <community_changelogs>` and include a :ref:`changelog fragment <changelogs_how_to>` if appropriate.
+
+Contributing documentation
+==========================
+
+Additions to the collection documentation are very welcome! We have three primary types of documentation, each with their own syntax and rules.
+
+Module and plugin documentation
+-------------------------------
+
+This type of documentation gets generated from structured YAML, inside of a Python string. It is included in the same code that it's documenting, or in a separate Python file, such as a doc fragment. Please see the :ref:`module format and documentation guidance <developing_modules_documenting>` for more information.
+
+This type of documentation is tested with ``ansible-test sanity`` and full instructions are available on the :ref:`testing module documentation <testing_module_documentation>` page.
+
+Although we can't preview how the documentation will look for these, we can be reasonably assured the output is correct because of the highly structured nature and the validation via sanity tests.
```

Comment:
```suggestion
Although we can't preview how the documentation will look for these, we can be reasonably sure the output is correct because the documentation is highly structured validated using sanity tests.
```

Reply:
ugh, I deleted the `and` there by mistake . . . "highly structured and validated using . . . "

Reply:
oh nice, I can update your suggestion and then commit it